### PR TITLE
Documentation of CSV exports formats

### PR DESCRIPTION
**Pull request by jmleroux**

### CODE REVIEW

File: reference/import_export/formats/examples-export/category.csv [hunk 1]
```diff
@@ -0,0 +1,141 @@
+code;parent;label-de_DE;label-en_US;label-fr_FR
+master;;Hauptkatalog;"Master catalog";"Catalogue principal"
+tvs_projectors;master;"TVs und projectoren";"TVs and projectors";"Téléviseurs et projecteurs"
+pc_monitors;tvs_projectors;"PC Monitoren";"PC Monitors";"Moniteurs PC"
+led_tvs;tvs_projectors;"TVs LED";"LED TVs";"Télévisions LED"
+cameras;master;Cameras;Cameras;Caméras
+digital_cameras;cameras;"Digitale Kameras";"Digital cameras";"Caméras digitales"
+camcorders;cameras;"Digitale Videokameras";Camcorders;"Caméscopes numériques"
+webcams;cameras;Webcams;Webcams;Webcams
```
Comment (fitn) [inline reference/import_export/formats/examples-export/category.csv]: Maybe you could remove the rest of this file

**Comment by fitn** [comment]
Just a note to keep in mind.
The directory name is "examples-export". It should be "examples" as it works for import AND export :)

**Comment by nidup** [comment]
nice PR don't understand where it will appears in the doc?

ping @Delphineray

**Comment by fitn** [comment]
@nidup The link with the current doc is not done yet. We would know that before merging it.

**Comment by Delphineray** [comment]
Yes, I've seen the PR, a great beginning to improve our documentation on our connector!

File: reference/import_export/formats/association_type.rst [hunk 2]
```diff
@@ -0,0 +1,15 @@
+Association types data structure
+================================
+
+Data structure
```
Comment (rybus) [inline reference/import_export/formats/association_type.rst]: You should merge title and subtitle, as you only have one sub part

File: reference/import_export/formats/attribute.rst [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+Data structure
+--------------
+The import/export file follows this data structure:
+
+- **code** (required): the attribute code.
+- **type** (required): the Akeneo PIM attribute type.
+- **localizable** (required): boolean field (0 or 1) defining if the attribute is localizable or not.
+- **scopable** (required): boolean field (0 or 1) defining if the attribute is scopable or not.
+- **group** (required): the attribute group of the attribute. Default could be "other".
```
Comment (rybus) [inline reference/import_export/formats/attribute.rst]: Default could be "other" => Default value **should**, **is**, **must be** would be better

File: reference/import_export/formats/attribute.rst [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+- **label-<locale_code>**: each label in a distinct column (See :doc:`localized-labels`).
+- **unique**: boolean field (0 or 1) defining if the value of this attribute is unique. Its value should be 1 for the `pim_catalog_identifier`.
+- **useable_as_grid_filter**: define if the attribute could be used as a filter in Akeneo PIM/
```
Comment (rybus) [inline reference/import_export/formats/attribute.rst]: Defines
Useless / at the end of the sentence

File: reference/import_export/formats/attribute.rst [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+  File example:
+
+    .. literalinclude:: examples/attribute.csv
+
+
+Attribute types
+---------------
+
+Here is the description of the attribute types:
```
Comment (rybus) [inline reference/import_export/formats/attribute.rst]: useless sentence

File: reference/import_export/formats/category.rst [hunk 6]
```diff
@@ -0,0 +1,16 @@
+Category data structure
+=======================
+
+Data structure
+--------------
+The import/export file follow this data structure:
```
Comment (rybus) [inline reference/import_export/formats/category.rst]: follow**s**

File: reference/import_export/formats/category.rst [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+
+- **code** (required): the category code
+- **parent**: code of the parent category. Blank if it is a root category.
```
Comment (rybus) [inline reference/import_export/formats/category.rst]: Leave blank if this is a root category

File: reference/import_export/formats/family.rst [hunk 8]
```diff
@@ -0,0 +1,18 @@
...
+=====================
+
+Data structure
+--------------
+The import/export file follows this data structure:
+
+- **code** (required): the family code
+- **attributes**: the list of the family attribute codes separated with a comma
+- **label-<locale_code>**: each label in a distinct column (See :doc:`localized-labels`)
+- **attribute_as_label**: the attribute which will be used by products of this family when they are displayed
```
Comment (rybus) [inline reference/import_export/formats/family.rst]: Attribute which value will be used as product label on product datagrid

File: reference/import_export/formats/product.rst [hunk 9]
```diff
@@ -0,0 +1,33 @@
+Product CSV data structure
+==========================
+
+Products are exported in a CSV file with the following fields:
+
+Attributes fields can be one of these:
```
Comment (rybus) [inline reference/import_export/formats/product.rst]: You should add all fields: sku is required, but you can also add categories (comma separated) and enabled

File: reference/import_export/formats/scopable-labels.rst [hunk 10]
```diff
@@ -0,0 +1,15 @@
+Scopable labels
+===============
+
+In the case of product export, all localized and/or scopable labels follow the same rule in the import/export:
+
+{code}-{scope}-{locale}
+
+Thus, a `foo` label available in ecommerce and print channels and in fr_FR, en_US and de_DE locales
+will be exported in six columns:
```
Comment (rybus) [inline reference/import_export/formats/scopable-labels.rst]: And what if my attribute is just scopable and not localizable ? You should explain it so people don't think scopable = localizable.

**Comment by rybus** [comment]
Be careful with "distinct" when writing "In a distinct column". I think "Dedicated" would be more relevant. But it's up to you :) Thanks for this PR!

File: reference/import_export/formats/product.rst [hunk 11]
```diff
@@ -0,0 +1,33 @@
...
+- **sku** (required): Identifier attribute (SKU is the common code)
+- **clothing_size**: size option code for a simple select option
+- **description-en_US-mobile**: localizable and scopable attribute en_US for channel mobile
+- **description-fr_FR-mobile**: localizable and scopable attribute fr_FR for channel mobile
+- **name-fr_FR**: localizable attribute fr_FR
+- **name-en_US**: localizable attribute en_US
+- **price-EUR**: numeric price value for Euro
+- **price-USD**: numeric price value for US Dollar
+- **tshirt_style**: list of option codes for a multi select option
```
Comment (ronanguilloux) [inline reference/import_export/formats/product.rst]: would change that in "option codes for a multi select option list"

File: reference/import_export/formats/product.rst [hunk 12]
```diff
@@ -0,0 +1,33 @@
...
+
+Associations fields:
+- **<association-type-code>-groups**: groups list for the <association-type-code> association type
+- **<association-type-code>-products**: list of products identifier in this association type
```
Comment (ronanguilloux) [inline reference/import_export/formats/product.rst]: would change that in "products identifier**s** list for this association type

**Comment by ronanguilloux** [comment]
Very helpful PR!

**Comment by rybus** [comment]
GTM

**Comment by nidup** [comment]
GTM

**Pull request merged**